app/Http/Controllers/Frontend/CheckoutController.php

```diff
@@ -166,8 +167,9 @@ public function showCheckout()
     public function checkout(CheckoutRequest $request)
     {
         $ordersData = $request->except('_token');
-        $ordersData['user_id'] = $request->user()->id;
-        $ordersData['status'] = 1;
+        $ordersData['user_id']    = $request->user()->id;
+        $ordersData['status']     = 1;
```

Comment:
This function Hoang Nguyen created.
status = 1: Orders are comfirmed.
status = 2: Orders was moved.
status = 3: Orders was shipped successfully.